frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.ts

```diff
@@ -0,0 +1,21 @@
+import type { OnInit } from '@angular/core';
+import { Input, Component } from '@angular/core';
+import { CanCaterFor } from 'digital-fuesim-manv-shared';
+
+export type CatererType = 'material' | 'personnel';
+
+@Component({
+    selector: 'app-cater-capacity',
+    templateUrl: './cater-capacity.component.html',
+    styleUrls: ['./cater-capacity.component.scss'],
+})
+export class CaterCapacityComponent implements OnInit {
+    @Input() canCaterFor?: CanCaterFor;
+    @Input() type?: CatererType
```

Comment:
If you don't want to make this optional, you can do:

```suggestion
    @Input() canCaterFor!: CanCaterFor;
    @Input() type!: CatererType
```

Sadly, this is one of the weaknesses of angulars type safety :/

Reply:
This doesn't mark it as required, but rather only expected it to be defined right?

Reply:
> This doesn't mark it as required, but rather only expected it to be defined right?

Exactly. 👍